src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Schemas/JSON/template.json

```diff
@@ -1287,13 +1291,13 @@
                 "oneOf": [
                   {
                     "type": "string",
-                    "enum": ["Windows", "Unix", "OSX"]
+                    "enum": ["Windows", "Linux", "OSX"]
                   },
                   {
                     "type": "array",
                     "items": {
                       "type": "string",
-                      "enum": ["Windows", "Unix", "OSX"]
+                      "enum": [ "Windows", "Linux", "OSX" ]
```

Comment:
Spaces were removed in schemastore repo as part of overall unification: https://github.com/SchemaStore/schemastore/pull/2255

Although we used to use different styling in our repo - I'd probably stick to styling from schemastore (as those might get enforced in their pipeline, unless they are already)